app/models/course.rb

```diff
@@ -276,7 +276,9 @@ def send_notification_to_accredited_body
     users.each do |user|
       CourseUpdateEmailMailer.course_update_email(
         self,
-        I18n.t("course.update_email.#{updated_attribute}"),
+        updated_attribute,
+        saved_changes[updated_attribute].first,
+        saved_changes[updated_attribute].second,
         user,
```

Comment:
I'm a bit confused by names of the args being passed into `course_update_email` and how these map to the method signature - `saved_changes[updated_attribute].first`, `saved_changes[updated_attribute].second`

Reply:
I agree, would it maybe make it easier to read if `#course_update_email` had named params. It also has the benefit of not having to know which order the params need to be

Reply:
I've updated it to use keyword arguments so hopefully that should be a bit clearer.